isso/js/app/config.js

```diff
@@ -81,4 +82,6 @@ config["langs"] = languages;
 delete config["lang"];
 delete config["default-lang"];
 
+config["page-author-hash"] = config["page-author-hash"].split(/[\s,]+/);
```

Comment:
Please comment in the code what this does, it's not very obvious to JS beginners at first glance
`// Split "page-author-hash" by comma and spaces into array`